.circleci/config.yml

```diff
@@ -56,6 +56,7 @@ workflows:
                 - node/macos
                 - node/windows
               node-version:
+                - 22.9.0
```

Comment:
question: do we plan to stop testing on older versions once we switch to Node 22?

Reply:
Yeah, I think once we perform the big bump we'll bump `engines` everywhere